### DISCUSSION TITLE
[Feature] Inner Album Sort Options

### DISCUSSION THREAD

**Discussion by XTREEMMAK** [immich-app/immich#5503] [Feature Request]
### The feature

Going inside any album does not allow for sort order. I like my albums to "tell a story" from oldest picture to newest at the end instead of the newest picture being on the top. It would be nice to be able to sort the inner albums in either direction any way we want independently.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by Pazu5512** [comment]
To add to what you said, I think that an album should indeed tell a story. 

Let's say an album has 3 photos, "A", "B" and "C". Sometimes, photo "C" I took 10 minutes later than photo "A" and photo "B" should actually be presented to the viewer in that specific order : "A", "C" and then "B".
Unfortunately, a simple filename or date sort order option doesn't allow that. 

tl;dr : the ideal solution is to allow the album owner to organize/position the photos however he wants inside an album, by a drag and drop, much like Google Photos does.

**Comment by bo0tzz** [comment]
Duplicate of #1689
